native/packages/react-native-bpk-component-text-input/src/BpkTextInput.js

```diff
@@ -54,13 +99,27 @@ class BpkTextInput extends Component {
     ]).start();
   }
 
-  onFocus() {
-    this.setState(() => ({ isFocused: true }), this.props.onFocus);
-  }
-
-  onBlur() {
-    this.setState(() => ({ isFocused: false }), this.props.onBlur);
-  }
+  onFocus = () => {
+    this.setState(
+      () => ({ isFocused: true }),
+      () => {
+        if (this.props.onFocus) {
+          this.props.onFocus();
+        }
+      },
+    );
+  };
+
+  onBlur = () => {
+    this.setState(
+      () => ({ isFocused: false }),
+      () => {
```

Comment:
Is just passing `this.props.onBlur` not a good idea anymore?

Reply:
Flow didn't like that because it could be null, so I had to change it

Reply:
That's strange though. I assumed that `setState` is fine with it being `null` too, but I guess it only supports `undefined`

Reply:
I mean, it works just fine in practice, but Flow doesn't like it. I could always just add a flowignore comment to make it go away, then we can keep the more succinct code.

Reply:
Nah leave it as is. The type in React is `?function`. https://github.com/facebook/react/blob/1c1f68e8dccbc84acfae66178493dbfb8ca7d1e1/src/isomorphic/modern/class/ReactComponent.js#L56

Reply:
Weird - you'd think that Flow would be able to infer that `setState`'s second argument is nullable and accept it. But yeah I'll just leave the code like this to appease it.

Reply:
Yeah it's strange. I would think it should handle `null` with this type signature